src/components/Card/CardsList.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import CardWrapper from '../UI/CardWrapper';
+
+interface CardsListProps {
+  cards: {
+    firstPage: string;
```

Comment:
**nitpick:** You've already declared such interface, let's reuse it

Reply:
Should I create separate file for interfaces and import them into my components? If yes where do I put them? 

> ./src  interfaces.ts ?

Reply:
As it's a business domain, you could do `src/data/card/types.ts`. But it's a small app, so it will be totally enough when you put it into: `src/types.ts` which will hold your all common types.